tutorials/build_systems/spack/spack_test.py

```diff
@@ -19,6 +19,7 @@ class BZip2SpackCheck(rfm.RegressionTest):
     @run_before('compile')
     def setup_build_system(self):
         self.build_system.environment = 'myenv'
+        self.build_system.specs = ['bzip2@1.0.6']
```

Comment:
If we change the tutorial example, we should make sure that the text is up to date. Also note that now an environment is not required anymore.